hardhat.config.ts

```diff
@@ -81,9 +83,7 @@ const config: HardhatUserConfig = {
     mainnet: {
       chainId: 1,
       url: MAINNET_RPC_URL,
-      accounts: {
-        mnemonic: MNEMONIC,
-      },
+      accounts: [DEPLOYER_PK],
```

Comment:
Do not use PK, use the mnemonic setup instead. Several scripts in the project depend on having multiple available accounts, and generally speaking it's better anyway.